packages/opentelemetry-exporter-collector/src/platform/node/sendSpans.ts

```diff
@@ -39,7 +38,7 @@ export function onInit(collectorExporter: CollectorExporter) {
     isShutDown: false,
     grpcSpansQueue: [],
   });
-  const serverAddress = removeProtocol(collectorExporter.url);
+  const serverAddress = collectorExporter.url;
```

Comment:
because of shared url people might put a protocol in front of url and it won't be working - hard to find out why. That's why I added `removeProtocol` here to avoid such situation.